<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>단타왕 김장타</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>감시시작</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="text">
         <string>감시정지</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="text">
         <string>종료</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>612</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget_jango">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <column>
         <property name="text">
          <string>종목코드</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>종목명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>잔고수량</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>매도가능</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>매입금액</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>장부가</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>현재가</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>수익률</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
